test/test_meta.py

```diff
@@ -329,6 +329,7 @@ def test_tensor_outlives_converter(self):
     aten.prelu.default,
     aten.special_xlog1py.default,
     aten.xlogy.Tensor,
+    aten.nll_loss2d_forward.default,
```

Comment:
Strides don't match.

Reply:
It previously matched, but this PR changed it to not match?

Reply:
From what I understood from the xfails, previously all meta tests for them were disabled. Now all work except for this one.